OrrRock/OrrRock/Utils/ColorPickerView.swift

```diff
@@ -14,12 +14,15 @@ protocol ColorPickerViewDelegate {
 
 class ColorPickerView: UIView{
     //피커뷰의 고정적으로 쓰이는 넓이
-    private var pickerWidth = 45
+    private var pickerWidth = 0.0
+    //피커뷰의 고정적으로 쓰이는 넓이
+    private var backgroundViewWidth = 0.0
```

Comment:
p4;
```suggestion
    private var pickerWidth: CGFloat = 0.0
    private var backgroundViewWidth: CGFloat = 0.0
```
타입을 명시하면 좋을 것 같아요!